packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/Stubs.m

```diff
@@ -271,6 +271,14 @@ - (NSData *)getReceiptData:(NSURL *)url error:(NSError **)error {
   return [[NSData alloc] initWithBase64EncodedString:originalString options:kNilOptions];
 }
 
+- (NSURL *)receiptURL {
```

Comment:
it's not a good practice for the test class to subclass from the real class. Can you add a TODO here to make both test and real classes to conform to a protocol?

Reply:
Sure. actually, this entire file is actually like this, where the stubs are extended from real classes. I will try to remove this file when we move the tests to swift.